src/inputs/TextFieldBase.tsx

```diff
@@ -255,6 +275,7 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
                   ...fieldStyles.input,
                   ...(inputProps.disabled ? fieldStyles.disabled : {}),
                   ...(showHover ? fieldStyles.hover : {}),
+                  ...(unfocusedPlaceholder && !isFocused && visuallyHidden),
```

Comment:
**question:** Will these return `false` ever? I'm kind of surprised we can do `...(false)` 🤔 I'm probably missing something. If it works, it works! 🎉

Reply:
yes it does, updated